update service scripts and git actions

AndyForWork opened the pull request:
https://github.com/SkoltechSummerCamp/5Gst/issues/34

AndyForWork committed:
Update balancer_migrations.yml

AndyForWork committed:
Update balancer_api_codegen.yml

AndyForWork committed:
Create balancer_build_from_fork.yml

AndyForWork committed:
Create service_build_from_fork.yml

AndyForWork committed:
Update service_build_from_fork.yml

AndyForWork committed:
Update balancer_build_docker.yaml

AndyForWork committed:
Update service_build_docker.yml

AdvancerMan commented on .github/workflows/balancer_build_from_fork.yml:
```diff
@@ -0,0 +1,19 @@
+ame: "[Balancer] build image from fork"
```
from fork скорее неправильно. Лучше просто "build docker image"

AdvancerMan replied:
В смысле pr может быть необязатеольно из форка

AdvancerMan commented on .github/workflows/balancer_build_from_fork.yml:
```diff
@@ -0,0 +1,19 @@
+      - uses: actions/checkout@v2
+        with:
+          token: ${{secrets.PERSONAL_ACCESS_TOKEN_GITHUB }}
+      - name: build from fork
```
from fork тоже не нужен

AdvancerMan commented on .github/workflows/service_build_from_fork.yml:
```diff
@@ -0,0 +1,92 @@
+          path: service/gh-artifacts
+      - name: build image
+        run: |
+          git submodule init
+          git submodule update
```
это можно сделать в actions/checkout

AdvancerMan replied:
Вроде recursive: true, что то такое

AdvancerMan replied:
```
      - uses: actions/checkout@v2
        with:
          submodules: recursive
```

AndyForWork committed:
Update balancer_build_from_fork.yml

AndyForWork committed:
Update balancer_build_from_fork.yml

AndyForWork committed:
Update service_build_from_fork.yml

AndyForWork committed:
Rename service_build_from_fork.yml to service_build_docker_after_pr.yml

AndyForWork committed:
Rename balancer_build_from_fork.yml to balancer_build_docker_after_pr.yml

AndyForWork committed:
Update balancer_build_docker_after_pr.yml

AndyForWork committed:
Update build-iperf.sh

AndyForWork commented:
https://github.com/SkoltechSummerCamp/5Gst/issues/9

AndyForWork committed:
Update build-iperf.sh

AndyForWork committed:
Update build-iperf.sh

AndyForWork committed:
Update build-iperf.sh

AndyForWork committed:
Update build-iperf.sh

AndyForWork committed:
Merge pull request #2 from SkoltechSummerCamp/develop

Add swagger codegen for service (#65)

AndyForWork committed:
Update CI.yml

Create CI.yml

Update service_api_codegen.yaml

Update CI.yml

Update CI.yml

Update balancer_api_codegen.yml

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Generate balancer client using swagger-codegen

Update CI.yml

Generate balancer client using swagger-codegen

Delete application_build.yml

Delete balancer_api_codegen.yml

Delete balancer_build_docker.yaml

Delete balancer_build_docker_after_pr.yml

Delete balancer_django_tests.yaml

Delete balancer_migrations.yml

Delete service_api_codegen.yaml

Delete service_build_docker.yml

Delete service_build_docker_after_pr.yml

Generate balancer client using swagger-codegen

Update CI.yml

Generate balancer client using swagger-codegen

AndyForWork commented:
https://github.com/SkoltechSummerCamp/5Gst/issues/54

AndyForWork committed:
Generate balancer client using swagger-codegen
